packages/core/src/configuration.ts

```diff
@@ -170,11 +173,30 @@ const axisChart: AxisChartOptions = Tools.merge({}, chart, {
 	ruler,
 	zoomBar: {
 		zoomRatio: 0.4,
+		minZoomRatio: 0.01,
 		top: {
 			enabled: false,
 			type: ZoomBarTypes.GRAPH_VIEW
 		}
-	} as ZoomBarsOptions
+	} as ZoomBarsOptions,
+	toolbar: {
+		enabled: false,
+		maxIcons: 3,
+		controlsInOrder: [
```

Comment:
```suggestion
		controls: [
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/791/commits/f6a4a5976a3106e01e7907c24c082647153b9858.